src/pages/instances/InstanceDetail.tsx

```diff
@@ -64,9 +58,15 @@ const InstanceDetail: FC = () => {
       }
       contentClassName="detail-page"
     >
-      <NotificationRow />
       {isLoading && <Loader text="Loading instance details..." />}
-      {!isLoading && !instance && <>Loading instance failed</>}
+      {!isLoading && !instance && !error && <>Loading instance failed</>}
+      {error && (
+        <Strip>
+          <Notification severity="negative" title="Error">
```

Comment:
`titleElement` of the `Notification` should match the heading structure of the page. I think h3 is the right one here.

Reply:
true, `h3` seems appropriate, although it is slightly strange that `h3` is appearing above the `h2` elements on this page. I think this can be included in the scope of the a11y audit issue rather since there are many places in the app where `Notification` is shown